isis/scripts/isis3VarInit.py

```diff
@@ -79,19 +85,43 @@ def mkdir( p ):
 mkdir( deactivate_dir )
 
 # Write the files that manage the ISIS3 environments:
-activate_vars   =   activate_dir+'/env_vars.sh'
-deactivate_vars = deactivate_dir+'/env_vars.sh'
-
-with open( activate_vars, mode='w' ) as a:
-    a.write('#!/bin/sh\n')
-    a.write('export ISISROOT='+      os.environ['CONDA_PREFIX']+'\n')
-    a.write('export ISIS3DATA='+     args.data_dir +'\n')
-    a.write('export ISIS3TESTDATA='+ args.test_dir +'\n')
-print( 'Wrote '+activate_vars )
-
-with open( deactivate_vars, mode='w' ) as d:
-    d.write('#!/bin/sh\n')
-    d.write('unset ISISROOT\n')
-    d.write('unset ISIS3DATA\n')
-    d.write('unset ISIS3TESTDATA\n')
-print( 'Wrote '+deactivate_vars )
+activate_vars_sh   =   activate_dir+'/env_vars.sh'
+deactivate_vars_sh = deactivate_dir+'/env_vars.sh'
+activate_vars_csh   =   activate_dir+'/env_vars.csh'
+deactivate_vars_csh = deactivate_dir+'/env_vars.csh'
+
+with open( activate_vars_sh, mode='w' ) as a:
+    script = """#!/bin/sh
+export ISISROOT={}
+export ISIS3DATA={}
+export ISIS3TESTDATA={}
+""".format(os.environ['CONDA_PREFIX'], args.data_dir, args.test_dir)
+    a.write(script)
+print( 'Wrote '+activate_vars_sh )
+
+with open( deactivate_vars_sh, mode='w' ) as d:
+    script = """#!/bin/sh
+unset ISISROOT
+unset ISIS3DATA
+unset ISIS3TESTDATA
+"""
+    d.write(script)
+print( 'Wrote '+deactivate_vars_sh )
+
+with open( activate_vars_csh, mode='w' ) as a:
+    script = """#!/bin/csh
+setenv ISISROOT {}
+setenv ISIS3DATA {}
+setenv ISIS3TESTDATA {}
+""".format(os.environ['CONDA_PREFIX'], args.data_dir, args.test_dir)
+    a.write(script)
+print( 'Wrote '+activate_vars_csh )
+
+with open( deactivate_vars_csh, mode='w' ) as d:
+    script = """#!/bin/sh
+unsetenv ISISROOT
+unsetenv ISIS3DATA
+unsetenv ISIS3TESTDATA
+"""
```

Comment:
These strings could be defined at the top of the file or right here. I'm not sure where the better place is, but this is closest to where they are first used.